### PR TITLE
Removed references to PHP 5, 7.0

### PR DESCRIPTION
**Pull request by saundefined**
PDO and PCRE sections

### CODE REVIEW

File: reference/pcre/pattern.syntax.xml [hunk 1]
```diff
@@ -1277,7 +1277,7 @@
   <para>
    In UTF-8 mode, characters with values greater than 128 do not match any
    of the POSIX character classes.
-   As of PHP 5.3.0 and libpcre 8.10 some character classes are changed to use
+   As libpcre 8.10 some character classes are changed to use
```
Comment (cmb69) [inline reference/pcre/pattern.syntax.xml:1280]:
```suggestion
   As of libpcre 8.10 some character classes are changed to use
```

File: reference/pdo_sqlsrv/configure.xml [hunk 2]
```diff
@@ -5,17 +5,12 @@
  <para>
   The PDO_SQLSRV extension is enabled by adding appropriate DLL file to your PHP 
   extension directory and the corresponding entry to the &php.ini; file. The 
-  PDO_SQLSRV download comes 8 driver files, four of which are for PDO support. 
-  If you are running non-thread-safe PHP (PHP 5.3), use the php_pdo_sqlsrv_53_nts.dll 
-  file. (You should use a non-thread-safe version if you are using IIS as your 
-  web server). If you are running thread-safe PHP, use the php_pdo_sqlsrv_53_ts.dll 
-  file. Similarly for PHP 5.4, use the php_pdo_sqlsrv_54_nts.dll or php_pdo_sqlsrv_54_ts.dll
-  depending on whether your PHP installation is non-thread-safe or thread-safe. 
+  PDO_SQLSRV download comes 8 driver files, four of which are for PDO support.
```
Comment (cmb69) [inline reference/pdo_sqlsrv/configure.xml:8]:
```suggestion
  PDO_SQLSRV download comes with 8 driver files, four of which are for PDO support.
```

File: reference/pdo_sqlsrv/configure.xml [hunk 3]
```diff
@@ -5,17 +5,12 @@
...
  </para>
  <para>
   The most recent version of the driver is available for download here:
-  <link xlink:href="&url.sqlsrv;">SQLSRV download</link>. If you need support 
-  for PHP 5.2 and/or PHP compiled with VC6, use the 2.0 release of the driver: 
+  <link xlink:href="&url.sqlsrv;">SQLSRV download</link>. 
+  If PHP compiled with VC6, use the 2.0 release of the driver: 
   <link xlink:href="&url.sqlsrv.v2driverdownload;">SQLSRV 2.0 download</link>.
```
Comment (cmb69) [inline reference/pdo_sqlsrv/configure.xml:14]: This info is no longer relevant; VC6 was used prior to PHP 5.3.0, so:
```suggestion
```

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you! Generally good, but there are a few issues.

**Comment by saundefined** [comment]
Thank you! Fixed

**Pull request merged**